plugins/prerequisite_stages/app/views/samson_prerequisite_stages/_stage_show.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if prerequisite_stages = @stage.prerequisite_stages.presence %>
```

Comment:
I think this will do 2 queries since it's .empty + .to_a

Reply:
Tested it out in rails console and it looks like it just does one select